index.php

```diff
@@ -1,7 +1,9 @@
 <?php
+require_once('app/init.php');
+//require(__DIR__ . '/vendor/autoload.php');
 
-use App\Core\Dispatcher;
+use App\Core\Bootstrap as Bootstrap;
+use App\Core\Router as Router;
```

Comment:
You do not need the `as` statements as it is redundant here, so please remove them too